projects/ui/src/lib/components/po-table/enums/po-table-row-template-arrow-direction.enum.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @usedBy PoTableComponent
```

Comment:
```suggestion
 * @usedBy PoTableRowTemplateDirective
```

Dessa forma aparece no Template e nao no Componente